tests/ci/fast_test_check.py

```diff
@@ -93,7 +93,27 @@ def process_results(result_folder):
     gh = Github(get_best_robot_token())
 
     rerun_helper = RerunHelper(gh, pr_info, NAME)
-    if rerun_helper.is_already_finished_by_status():
+    finished_by_status = rerun_helper.get_finished_status()
+    if finished_by_status in ["success", "failure"]:
+        if (
+            SKIP_SIMPLE_CHECK_LABEL not in pr_info.labels
+            and finished_by_status != "success"
+            and (
+                FORCE_TESTS_LABEL not in pr_info.labels or finished_by_status == "error"
+            )
+        ):
+            url = (
+                f"{GITHUB_SERVER_URL}/{GITHUB_REPOSITORY}/"
+                "blob/master/.github/PULL_REQUEST_TEMPLATE.md?plain=1"
+            )
+            commit = get_commit(gh, pr_info.sha)
+            commit.create_status(
+                context="Simple Check",
+                description=f"{NAME} failed",
+                state="failed",
+                target_url=url,
+            )
```

Comment:
It looks to me like the whole thing will be used quite wide. We should move it to tests/ci/commit_status_helper.py

Reply:
if conditions are not universal, and as I see we already have `post_commit_status` there, so I see the only thing can be done is this:  
```
    finished_by_status = rerun_helper.get_finished_status()
    if finished_by_status in ["success", "failure"]:
        if (
            SKIP_SIMPLE_CHECK_LABEL not in pr_info.labels
            and finished_by_status != "success"
            and (
                FORCE_TESTS_LABEL not in pr_info.labels or finished_by_status == "error"
            )
        ):
            url = (
                f"{GITHUB_SERVER_URL}/{GITHUB_REPOSITORY}/"
                "blob/master/.github/PULL_REQUEST_TEMPLATE.md?plain=1"
            )
            post_commit_status(gh, pr_info.sha, "Simple Check", f"{NAME} failed", "failed", url)
```